### DISCUSSION TITLE
Translations Pluralization

### DISCUSSION THREAD

**Discussion by lukas-frey** [filamentphp/filament#4628] [General]
I'm currently adding missing translations to the czech locale and I came across an issue while translating `select.max_items_message` in the filament/forms package (This bug doesn't apply only to this specific key).

I can't properly pluralize the translations, since it uses the standard `__()` syntax instead of `trans_choice()` so it can be translated for different plural numbers.

**Comment by zepfietje** [comment]
Can you PR the required change? Then we will review and merge it!

**Reply by lukas-frey** [reply]
> Can you PR the required change? Then we will review and merge it!

Sure! I don't know where everywhere it has to change, so I will make a PR to change only those few translations I came across today at first.

I found a few wrong translations anyways, so when I get the time to go through each translation one by one I will also check for additional missing pluralizations and make another PR.

**Reply by zepfietje** [reply]
Perfect! Totally fine to PR a batch first.
